.teamcity/_self/projects/WebApp.kt

```diff
@@ -637,7 +637,7 @@ fun playwrightPrBuildType( targetDevice: String, buildUuid: String ): BuildType
 					export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL%/}\"}"
 					export DEBUG=pw:api
 
-					xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%E2E_WORKERS% --group=calypso-pr
+					for i in {1..100}; do xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%E2E_WORKERS% specs/specs-playwright/wp-media__upload; done
```

Comment:
I assume this will be reverted before merging?

Reply:
Yes, this is only for stress testing purposes.

Reply:
Removed in 94ca9bd42a5816208ce94c9248a11e7ec69a6d57.